plaso/parsers/google_logging.py

```diff
@@ -0,0 +1,262 @@
+"""Parser for Google-formatted log files."""
+
+from __future__ import unicode_literals
+
+import re
+
+from dfdatetime import time_elements as dfdatetime_time_elements
+
+import pyparsing
+
+from plaso.containers import events
+from plaso.containers import time_events
+from plaso.lib import errors
+from plaso.lib import definitions
+from plaso.parsers import manager
+from plaso.parsers import text_parser
+
+
+class GoogleLogEventData(events.EventData):
+  """Google log file event data.
+
+  See: https://github.com/google/glog
+
+  Attributes:
+    file_name (str): the name of the source file that logged the message.
+    line_number (int): the line number in the source file where the logging
+      statement is.
+    message (str): the log message.
+    priority (str): the priority of the message - I, W, E or F.
+    thread_id (int): the ID of the thread that recorded the message.
+  """
+
+  DATA_TYPE = 'googlelog:log'
+
+  def __init__(self, data_type=DATA_TYPE):
+    """Initializes the event object.
+
+    Args:
+      data_type (Optional[str]): event data type indicator.
+    """
+    super(GoogleLogEventData, self).__init__(data_type=data_type)
+    self.file_name = None
+    self.line_number = None
+    self.message = None
+    self.priority = None
+    self.thread_id = None
+
+
+class GoogleLogParser(text_parser.PyparsingMultiLineTextParser):
+  """A parser for Google log formatted files.
+
+  See https://google.io/docs/python/guides/logging
+  """
+  NAME = 'googlelog'
+
+  DESCRIPTION = 'Parser for handling Google log formatted files.'
+
+  # PyParsing components used to construct grammars for parsing lines.
+  _PYPARSING_COMPONENTS = {
+      'priority':
```

Comment:
Please put `(` on the end of the line so you don't need 1 additional space for continuation indentation, which is also more consistent with the other definitions in this dict.